packages/@webex/plugin-meetings/src/breakouts/README.md

```diff
@@ -0,0 +1,191 @@
+# Breakout Sessions
+
+Breakout sessions are like meetings within meetings. They have separate roster lists and titles. Participants can move between breakouts and the main session.
+
+### Enabling
+
+To enable breakouts the `breakoutsSupported` parameter must be passed to the meeting join method.
+
+### Structure
+
+The current breakout session is accessible as follows:
+```javascript
+breakouts.currentBreakoutSession;
+```
+
+Other breakout sessions are available in the breakouts collection. These are breakouts that you known about. If you are an attendee, you may not know about all the breakout sessions in the meeting.
+```javascript
+breakouts.breakouts;
+```
+
+note: The `currentBreakoutSession` is NOT a member of the `breakouts.breakouts` collection.
+
+Some information about the state of breakouts is available on the `meeting.breakouts` object itself
+
+```javascript
+
+// The number of seconds after ending breakouts before the participants are returned to the main session 
+breakouts.delayCloseTime
+
+// Whether the participant is allowed back to the main session
+breakouts.allowBackToMain
+
+```
+
+### Attendee functionality
+
+The normal meeting members are updated when you move in and out of a breakout session.
+
+The following are methods available to attendees of a meeting:
+
+```javascript
+// Join a breakout
+// This can be used when the participant is in the main session or in a different breakout session
+breakout.join()
+
+// Leave a breakout
+// This will cause the participant to return to the main session
+breakouts.currentBreakoutSession.leave()
+
+// Ask for help when in a breakout session. 10 second cooldown.
+breakout.currentBreakoutSession.askForHelp()
+```
+
+#### Members
+
+The current session members are available on the meeting object as normal
```

Comment:
Maybe rephrase this, hard to understand what you mean.

Reply:
I mean the whole paragrah/s? Can't tell

Reply:
I've updated this paragraph. Hopefully it is clearer now